src/pages/sign-up-success-page/index.js

```diff
@@ -1 +1,13 @@
 import './index.scss';
+
+const pageLocalStorageKey = 'anketa/last';
+
+document.querySelector('.content .email').innerHTML = `(${localStorage.getItem(
+  `${pageLocalStorageKey}.student.email`
+)})`;
+
+const pageVisitedKey = `${pageLocalStorageKey}.isVisited`;
+
+!localStorage.getItem(pageVisitedKey)
+  ? localStorage.setItem(pageVisitedKey, 'true')
+  : (location.href = '/');
```

Comment:
handle languages, now all other languages will redirect to default(ukrainian)

Reply:
fixed in [redirect to correct home page depending on language](https://github.com/shpp/programming.org.ua/pull/106/commits/80913d4008142782922b1b4a7d185e06c545d627)